### PR TITLE
Update rubocop dependency to 0.47.1

### PR DESCRIPTION
**Pull request by vitorgalvao**
Same as Homebrew.

@jawshooah Mind cutting a new release when you have the chance? This might start breaking soon for people.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
@jawshooah Mind cutting a new release when you have the chance? This might start breaking soon for people.

**Comment by reitermarkus** [comment]
`Lint/Eval` has to be changed to `Security/Eval`.

File: Gemfile [hunk 1]
```diff
@@ -29,5 +29,5 @@ end
 local_gemfile = 'Gemfile.local'
```
Comment (reitermarkus) [inline Gemfile]: @jawshooah, is this even needed? What is this used for?

Comment (jawshooah) [inline Gemfile]: No, not really. I believe it was part of a gem template that I used as a basis for this repo, and I just never got rid of it.

**Comment by jawshooah** [comment]
Sorry for the delay on this, guys. I'm currently on my phone on a plane, but I'll merge and cut a new release when I get home.

**Comment by reitermarkus** [comment]
@vitorgalvao, seem's like there are some other breaking changes.

The `Gemfile` has to be sorted alphabetically, and there seems to be a method `def_matcher` which we are calling which doesn't exist in `rubocop 0.47.1`.

**Comment by reitermarkus** [comment]
`RuboCop::Node` in `rubocop/cask/extend/node` has to be renamed/refactored to `RuboCop::AST::Node`.

**Comment by vitorgalvao** [comment]
@reitermarkus Might be faster if you do it. I’m mostly following your instructions.

**Comment by vitorgalvao** [comment]
@reitermarkus Thank you.

**Pull request closed without merging**